### PR TITLE
PIN-4283 It is possibile to create a fresh new descriptor with a non-existent attribute

### PR DESCRIPTION
**Pull request by nttdata-rtorsoli**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -69,6 +69,12 @@ object Dependencies {
   private[this] object pagopa {
     lazy val namespace = "it.pagopa"
 
+    lazy val attributeRegistryManagementClient =
+      namespace %% "interop-be-attribute-registry-management-client" % attributeRegistryManagementVersion
```
Comment (galales) [inline project/Dependencies.scala]: not required, in this PR we are reading from the read model and not calling the management
```suggestion
```

File: project/Dependencies.scala [hunk 2]
```diff
@@ -110,6 +116,8 @@ object Dependencies {
       logback.classic                          % Compile,
       akka.slf4j                               % Compile,
       resilience4j.rateLimiter                 % Compile,
+      pagopa.attributeRegistryManagementClient % Compile,
```
Comment (galales) [inline project/Dependencies.scala]:
```suggestion
```

File: src/main/resources/application-standalone.conf [hunk 3]
```diff
@@ -25,6 +25,7 @@ catalog-process {
   services {
     catalog-management = "https://gateway.interop.pdnd.dev/catalog-management/0.1"
     authorization-management = "https://gateway.interop.pdnd.dev/key-management/0.1"
+    attribute-registry-management =  "https://gateway.interop.pdnd.dev/attribute-regsitry-management/0.1"
```
Comment (galales) [inline src/main/resources/application-standalone.conf]:
```suggestion
```

File: src/main/resources/application.conf [hunk 4]
```diff
@@ -25,6 +25,7 @@ catalog-process {
   services {
     catalog-management = ${CATALOG_MANAGEMENT_URL}
     authorization-management = ${AUTHORIZATION_MANAGEMENT_URL}
+    attribute-registry-management = ${ATTRIBUTE_REGISTRY_MANAGEMENT_URL}
```
Comment (galales) [inline src/main/resources/application.conf]:
```suggestion
```

File: src/main/scala/it/pagopa/interop/catalogprocess/common/system/ApplicationConfiguration.scala [hunk 5]
```diff
@@ -8,8 +8,10 @@ object ApplicationConfiguration {
 
   val serverPort: Int = config.getInt("catalog-process.port")
 
-  val catalogManagementUrl: String       = config.getString("catalog-process.services.catalog-management")
-  val authorizationManagementUrl: String = config.getString("catalog-process.services.authorization-management")
+  val catalogManagementUrl: String           = config.getString("catalog-process.services.catalog-management")
+  val authorizationManagementUrl: String     = config.getString("catalog-process.services.authorization-management")
+  val attributeRegistryManagementUrl: String =
+    config.getString("catalog-process.services.attribute-registry-management")
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/catalogprocess/common/system/ApplicationConfiguration.scala]:
```suggestion
```

**Review by beetlecrunch — APPROVED** [review]
I approve, but @galales comments must be applied.

**Review by galales — APPROVED** [review]

**Pull request merged**